crates/holochain/src/core/workflow/sys_validation_workflow.rs

```diff
@@ -652,7 +973,7 @@ fn update_check(entry_update: &Update, original_action: &Action) -> SysValidatio
 pub struct SysValidationWorkspace {
     scratch: Option<SyncScratch>,
     authored_db: DbRead<DbKindAuthored>,
-    dht_db: DbRead<DbKindDht>,
+    dht_db: DbWrite<DbKindDht>,
```

Comment:
The DHT database was previously being fetched from the space rather than the workspace. The workflow has always has write access that way. However, the space is no longer passed into the workflow (or rather, it is, but it's deliberately hidden inside the incoming ops sender).